### ISSUE TITLE
Dependency on selenium-webdriver (~> 3.0)

### ISSUE THREAD

**Issue by martinjozef** [hashrocket/capybara-webmock#44]
selenium-webdriver 3 series last update was in December 2019.
https://rubygems.org/gems/selenium-webdriver/versions

Should capybara-webmock depend or at least allow usage of more recent release (~> 4.0)  of `selenium-webdriver` ?

Here is an example of unmet dependency trying to keep up with more recent releases
```The unmet dependencies are:
* selenium-webdriver (~> 3.0), depended upon capybara-webmock-0.6.0, unsatisfied by selenium-webdriver-4.1.0
```

**Comment by dillonhafer** [comment]
I don't mind merging a PR and cutting a new release if you would like to PR it, but you don't have to.

**Comment by dan-jensen** [comment]
@dillonhafer here's a PR: https://github.com/hashrocket/capybara-webmock/pull/45

**Comment by mvz** [comment]
@dillonhafer could you please cut a new release that includes this dependency update?

**Comment by pozabil** [comment]
@hashrocketeer Hello! I still can't install gem with bundler, cause it throws an error:
```
  In Gemfile:
    capybara-webmock (~> 0.6.0) was resolved to 0.6.0, which depends on
      selenium-webdriver (~> 3.0)

    webdrivers was resolved to 5.0.0, which depends on
      selenium-webdriver (~> 4.0)
```
